macOS/DuckDuckGo/Common/Utilities/UserDefaultsWrapper.swift

```diff
@@ -92,7 +92,7 @@ public struct UserDefaultsWrapper<T> {
         case launchToCustomHomePage = "preferences.startup.launch-to-custom-home-page"
         case customHomePageURL = "preferences.startup.customHomePageURL"
         case openFireWindowByDefault = "preferences.startup.open-fire-window-by-default"
-        case currentThemeName = "com.duckduckgo.macos.currentThemeNameKey"
+        case themeAppearance = "com.duckduckgo.macos.currentThemeNameKey"
```

Comment:
I’m preserving the original key string. Should we write a migration process? Are we okay with this change?

Feedback welcomed, thank you!